nerdlandbot/commands/random.py

```diff
@@ -0,0 +1,133 @@
+import os
+import discord
+import random
+import aiohttp
+import asyncio
+
+from discord.ext import commands
+from nerdlandbot.helpers.constants import WOMBATS_DIR_NAME, PONCHO_DIR_NAME
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.log import warn
+from nerdlandbot.helpers.constants import EIGHT_BALL_URL,DAD_JOKE_URL
+from nerdlandbot.helpers.channel import get_channel
+
+def pick_random_online_member(ctx: commands.Context) -> discord.Member:
+    """
+    Selects a random online member from the server.
+    :param ctx: The current context: (discord.ext.commands.Context)
+    :return: The chosen member. (discord.Member)
+    """
+
+    def is_online(user: discord.Member):
+        if user.status == discord.Status.online:
+            return True
+        elif user.status == discord.Status.idle:
+            return True
+        else:
+            return False
+
+    online_members = filter(is_online, ctx.guild.members)
+    return random.choice(list(online_members))
+
+
+class Random(commands.Cog, name="Random"):
+    def __init__(self, bot):
+        self.bot = bot
+        
+        current_directory = os.getcwd()
+        
+        if not os.path.exists(WOMBATS_DIR_NAME):
+            os.makedirs(WOMBATS_DIR_NAME)
+            warn(f"Directory {os.path.join(current_directory,WOMBATS_DIR_NAME)} is created, put some wombat pictures in it!" )
+            msg = translate("empty_wombat_list", await culture(ctx))
```

Comment:
don't think `ctx` is properly defined here already. Since the __init__ is called when registering the cog at startup of the bot and does not have context as an argument.

Reply:
Jup, it gave me this:
![image](https://user-images.githubusercontent.com/7468683/110256075-2f94c400-7f97-11eb-9a30-b70fabac7a6f.png)

Reply:
Damnit you are right, I should pay more attention...

Reply:
@SelleslaghGianni yes, because there should be no await in the init either

Reply:
> Damnit you are right, I should pay more attention...

or run the code before committing and pushing ;) (I've been there a couple times to learn it the hard way)